CMakeLists.txt

```diff
@@ -345,14 +298,10 @@ add_custom_target(
 )
 add_dependencies(celerity_runtime celerity_version)
 
-if(CELERITY_SYCL_IMPL STREQUAL "ComputeCpp")
-  target_link_libraries(celerity_runtime PUBLIC ComputeCpp::ComputeCpp)
-endif()
-
 set(DEVICE_SOURCES "")
 if(CELERITY_SYCL_IMPL STREQUAL "hipSYCL")
   # For hipSYCL we have to pass all source files into add_sycl_to_target
-  # Don't just do it in general (yields better ComputeCpp build performance)
+  # Don't just do it in general (could yield better DPC++ build performance?)
```

Comment:
Our hand-rolled `add_sycl_to_target` for DPC++ ignores `SOURCES`, so we could also just get rid of this entire block.

Reply:
Even better! Removed.